Student/Student/Routes.swift

```diff
@@ -203,6 +203,9 @@ let router = Router(routes: HelmManager.shared.routeHandlers([
     "/:context/:contextID/discussions/:discussionID": discussionViewController,
     "/:context/:contextID/discussion_topics/:discussionID": discussionViewController,
 
+    "/groups/:groupID/discussions/:discussionID": discussionViewController,
+    "/groups/:groupID/discussion_topics/:discussionID": discussionViewController,
+
```

Comment:
Are these necessary? Routing to groups works now so there's definitely a route for them already registered in this file. These should be it:
```
"/:context/:contextID/discussions/:discussionID"
"/:context/:contextID/discussion_topics/:discussionID"
```

Reply:
You are right, they are not needed.